From stern@rowland.harvard.edu Mon Mar  6 07:45:50 2006
Date: Mon, 6 Mar 2006 10:45:42 -0500 (EST)
From: Alan Stern <stern@rowland.harvard.edu>
To: Greg KH <greg@kroah.com>
cc: "Rodolfo Quesada Z." <rquesada@eml.cc>, Phil Dibowitz <phil@ipom.com>
Subject: USB: storage: new unusual_devs.h entry: Mitsumi 7in1 Card Reader
Message-ID: <Pine.LNX.4.44L0.0603061042190.5142-100000@iolanthe.rowland.org>

From: Rodolfo Quesada <rquesada@roqz.net>

This patch (as661) adds an unusual_devs entry for the Mitsumi 7in1 Card
Reader.

From: Rodolfo Quesada <rquesada@roqz.net>
Signed-off-by: Alan Stern <stern@rowland.harvard.edu>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/storage/unusual_devs.h |    7 +++++++
 1 file changed, 7 insertions(+)

--- gregkh-2.6.orig/drivers/usb/storage/unusual_devs.h
+++ gregkh-2.6/drivers/usb/storage/unusual_devs.h
@@ -62,6 +62,13 @@ UNUSUAL_DEV(  0x03ee, 0x6901, 0x0000, 0x
 		US_SC_DEVICE, US_PR_DEVICE, NULL,
 		US_FL_SINGLE_LUN ),
 
+/* Reported by Rodolfo Quesada <rquesada@roqz.net> */
+UNUSUAL_DEV(  0x03ee, 0x6906, 0x0003, 0x0003,
+		"VIA Technologies Inc.",
+		"Mitsumi multi cardreader",
+		US_SC_DEVICE, US_PR_DEVICE, NULL,
+		US_FL_IGNORE_RESIDUE ),
+
 UNUSUAL_DEV(  0x03f0, 0x0107, 0x0200, 0x0200, 
 		"HP",
 		"CD-Writer+",
